### ISSUE TITLE
cuf1 TF targets

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#1330]
from
https://github.com/pombase/curation/issues/22

I think there is probably  enough info to move the targets of cuf1 from BP onto MF?

Especially if they have the CuRE.

**Comment by Antonialock** [comment]
I don't think there is enough evidence from those older papers to make it a MF. However, they are already annotated to the MF from a newer publication.

<img width="663" alt="Screenshot 2020-01-16 at 11 02 11" src="https://user-images.githubusercontent.com/12727506/72519693-cb251b80-384f-11ea-9126-81d3d21fb78a.png">

I removed the BP extensions since they appear cluttering. I screenshotted them if you think they should be added back.

<img width="1112" alt="Screenshot 2020-01-16 at 11 04 16" src="https://user-images.githubusercontent.com/12727506/72519775-fb6cba00-384f-11ea-95f2-4762b2fea4d6.png">
